src/components/index.d.ts

```diff
@@ -1037,6 +1037,39 @@ export declare const NoraDrawer: {
   }
 }
 
+export declare const Accordion: {
+  ({
+    id,
+    expanded,
+    onToggle,
+    children,
+  }: {
+    id?: string
+    expanded: object
+    onToggle: any
+    children: React.ReactNode
+  }): JSX.Element
+  propTypes: {
+    props: any
+  }
+}
+export declare const AccordionSection: {
+  ({
+    index,
+    title,
+    renderToggle,
+    children,
+  }: {
+    index?: number
+    title: string
+    renderToggle?: any
+    children?: React.ReactNode
```

Comment:
In the component file, `children` seems to be a required `prop`.  I think it makes sense for `children` to be required?

Reply:
🤦‍♂ yup. fixing, thanks Steph